### PR TITLE
update transit widget to account for chicago bus schedule

### PR DESCRIPTION
**Pull request by priscillahuff91**
@artkay @biteknight Please review PR

### CODE REVIEW

File: dist/transit-widget.js [hunk 1]
```diff
@@ -11,9 +11,9 @@
 	 * Only gets called when we're using $('$el').transitWidget format
 	 */
 	var TransitWidget = function(el, customOptions) {
-		if(!customOptions.stopID) {
+		if(!customOptions.stopID || customOptions.dma == 514) {
```
Comment (artkay) [inline dist/transit-widget.js]: @priscillahuff91 let's remove `|| customOptions.dma == 514` from here - I believe this is outside of the scope of this PR.

File: src/transit-widget.js [hunk 2]
```diff
@@ -5,9 +5,9 @@
 	 * Only gets called when we're using $('$el').transitWidget format
 	 */
 	var TransitWidget = function(el, customOptions) {
-		if(!customOptions.stopID) {
+		if(!customOptions.stopID || customOptions.dma == 514) {
```
Comment (artkay) [inline src/transit-widget.js]: @priscillahuff91 same here

**Comment by priscillahuff91** [comment]
@artkay code has been removed

File: dist/transit-widget.js [hunk 3]
```diff
@@ -484,6 +499,27 @@
...
+				data.stopName = data.buses[0].stpnm;
+				
+				data.buses.forEach(function(bus){
+					// Handling how long till bus arrives
+					var arrive = bus.prdtm;
+				    var current = bus.tmstmp;
+				    
+				    bus.arrivalTime = arrive.substr(arrive.length - 5).replace(':','');
+					bus.currentTime = current.substr(current.length - 5).replace(':','');
+				    bus.time = bus.arrivalTime - bus.currentTime;
```
Comment (djskripta) [inline dist/transit-widget.js]: Spacing inconsistency

**Comment by priscillahuff91** [comment]
@djskripta It just looks like weird spacing in Github if you go to that line of code the spacing is correct

**Comment by artkay** [comment]
Probably it is space vs tabs issue.

**Comment by priscillahuff91** [comment]
@artkay @djskripta Let me know if that looks right to you

**Comment by artkay** [comment]
@priscillahuff91 looks good to me
